editor/modes/visual-editor/block-list.js

```diff
@@ -210,7 +210,7 @@ class VisualEditorBlockList extends Component {
 					type="text"
 					readOnly
 					className="editor-visual-editor__placeholder"
-					value={ ! blocks.length ? __( 'Write your story.' ) : __( 'Write…' ) }
+					value={ ! blocks.length ? __( 'Write your story' ) : __( 'Write…' ) }
```

Comment:
I'd be okay with an ellipsis instead of a period. But it's just double _ellipsi_ then...